Sources/ProjectDescription/DeploymentTargets.swift

```diff
@@ -2,11 +2,17 @@ import Foundation
 
 // MARK: - DeploymentTargets
 
+// A struct representing the minimum deployment versions for each platform.
```

Comment:
Don't you need 3 slashes (`///`) to show up in doc? Same everywhere else

Reply:
yup, thanks, missed this on initial push